test/onigumo_test.exs

```diff
@@ -24,12 +24,14 @@ defmodule OnigumoTest do
 
   @tag :tmp_dir
   test("load URL from file", %{tmp_dir: tmp_dir}) do
+    input_urls = [@url]
+
     path = Path.join(tmp_dir, @input_path)
-    content = @url <> "\n"
+    content = urls_input(input_urls)
```

Comment:
```suggestion
    content = prepare_content(input_urls)
```

Reply:
Commented in https://github.com/Glutexo/onigumo/pull/44#issuecomment-1029071593.